### DISCUSSION TITLE
Question about mujoco simulation on snow

### DISCUSSION THREAD

**Discussion by sanbens** [google-deepmind/mujoco#1523] [Asking for Help] [answered]
hello everyone, I try to simulate human skiing on mujoco, specially about snow plow. And my question is whether mujoco provide a similar material like snow, if not, then whether there is some tricks to simulate snowplow braking and generate  the corresponding friction.
I would appreciate  your help！

**Comment by yuvaltassa** [comment, marked as answer]
Look at issue #67, might be relevant. Otherwise I don't think this is easy to do (e.g. the accumulation of snow would be rather hard)
